### PR TITLE
shared_page: set network state to Local

### PR DESCRIPTION
**Pull request by valentinvanelslande**
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3619)

### CODE REVIEW

File: src/core/hle/shared_page.h [hunk 1]
```diff
@@ -60,7 +60,7 @@ struct SharedPageDef {
     DateTime date_time_1;                // 40
     u8 wifi_macaddr[6];                  // 60
     u8 wifi_link_level;                  // 66
-    u8 wifi_unknown2;                    // 67
+    u8 network_state;                    // 67
```
Comment (jroweboy) [inline src/core/hle/shared_page.h]: Make a NetworkState enum instead with all the known values from 3dbrew

**Comment by valentinvanelslande** [comment]
@B3n30 HOME Menu and System Settings uses this, but for System Settings Local and Enabled is the same

File: src/core/hle/shared_page.h (hunk 1 above)
Comment (baka0815) [reply src/core/hle/shared_page.h]: I'll second that. Avoid magic numbers as much as possible.

**Comment by wwylele** [comment]
Closing for inactivity.

**Pull request closed without merging**